src/components/Profile.jsx

```diff
@@ -19,14 +19,14 @@ export default class Profile extends React.Component {
         _id: studentId
       }
     })
-    .then((response) => {
-      if (response.data[0]) {
-        this.setState({ profileData: response.data[0] });
-      }
-    })
-    .catch((error) => {
-      console.log('getProfileData error', error);
-    });
+      .then((response) => {
```

Comment:
Ah, I see. That makes sense. I would maybe indent lines 18-21 as well so it looks more like a normal Promise structure.